grype/match/matches.go

```diff
@@ -1,57 +1,52 @@
-package result
+package match
 
 import (
-	"github.com/anchore/grype/grype/match"
 	"github.com/anchore/syft/syft/pkg"
 )
 
-// TODO: consider moving this to the pkg/match under matches.go
-
-// TODO: consider renaming to Matches
-
-type Result struct {
-	byPackage map[pkg.ID][]match.Match
+type Matches struct {
+	byPackage map[pkg.ID][]Match
 }
 
-func NewResult() Result {
-	return Result{
-		byPackage: make(map[pkg.ID][]match.Match),
+func NewResult() Matches {
```

Comment:
so this, I think, would need to change to `NewMatch()` or perhaps `NewMatches()` ?

Reply:
agreed, `NewMatches`